cryptkeeper_test.go

```diff
@@ -182,21 +183,21 @@ func TestCryptString(t *testing.T) {
 			if err == nil {
 				t.Fatalf("Scan of int64 should have errored")
 			} else if csGoofyType.String != "" {
-				t.Fatalf("Scan of int64 should have left String empty, got: %s", csGoofyType.String)
+				t.Fatalf("Scan of int64 should have left String empty, got: %v", csGoofyType.String)
 			}
 
 			var csError CryptString
 			err = csError.Scan(interface{}([]byte("some bad encrypt")))
 			if err == nil {
 				t.Fatalf("Scan of bad encryption should have errored")
 			} else if csError.String != "" {
-				t.Fatalf("Scan of bad encryption should have left String empty, got: %s", csError.String)
+				t.Fatalf("Scan of bad encryption should have left String empty, got: %v", csError.String)
 			}
 
 		})
 		t.Run("Value", func(t *testing.T) {
 			var cs CryptString
-			cs.String = "hello world!"
+			cs.NullString = sql.NullString{"hello world!", true}
```

Comment:
now that you've embedded the struct, is it possible to do the following:
```suggestion
			cs.String = "hello world!"
			cs.Valid = true
```